.github/workflows/pr.yml

```diff
@@ -0,0 +1,64 @@
+name: Pull Request Action
+on:
+  workflow_dispatch
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v3
+
+      - name: Set up Git
+        run: |
+          git config --global user.name "${{ github.actor }}"
+          git config --global user.email "${{ github.actor }}@users.noreply.github.com"
+      
+      - name: Install jq
+        run: sudo apt-get -y install jq
+      
+      - name: Update config params for partner projects
+        run: |
+          declare -a repoUrls=(
```

Comment:
Partner URLs should be dynamically fetched from https://github.com/ubiquity/devpool-directory/blob/development/projects.json